### ISSUE TITLE
MJML is very slow and slow in macOS Monterey and Ventura. 

### ISSUE THREAD

**Issue by vysakhjayadas** [mjmlio/mjml-app#385]
I am able to open the app, but the code and images take a while to download and I can not enter or exit a file without a delay.

Normally, the code and images display instantly and I'm able to flow smoothly between files.
My colleagues are also facing similar issue after they upgrade the macOS to Monterey and Ventura .
The app is working fine in Catalina devices. 

Please fix the issue ASAP. Since its affecting our work. 

As you already aware Catalina is no longer supported by Apple. So this will impact around 30 + users .

**Comment by howdytom** [comment]
Yeah, any plans to fully support Apple Silicon? Is MJML App still in active development?
